src/style.css

```diff
@@ -0,0 +1,96 @@
+:root {
```

Comment:
Make sure all these CSS also is matching the new sample preview matching the application